### ISSUE TITLE
GPL-nnn As a user when the destination plate is ready I would like to scan the barcode into the limber pipeline and decide whether it will be processed as a 384 or 96 well plate

### ISSUE THREAD

**Issue by stevieing** [sanger/General-Backlog-Items#77]
**User story**
When the destination plate is ready scan the barcode into the limber pipeline and at this stage the user will make the decision on whether it will be quad stamped or continue to be processed as a 96 well plate and then a library submission will be created automatically.

**Who are the primary contacts for this story**
e.g. John S (don't include surnames in public repos)

**Acceptance criteria**
To be considered successful the solution must allow:
- [ ] add a list of acceptance criteria here
- [ ] ...

**Dependencies**
This story is blocked by the following dependencies:
- #<issue_no.>
- sanger/<repo>#<issue_no.>

**Additional context**

**Comment by KatyTaylor** [comment]
There were questions in our call around whether the lab staff will be happy to use the existing quad-stamping page in Sequencescape, or whether we should add this step to the 384 Limber pipeline.

**Comment by stevieing** [comment]
> There were questions in our call around whether the lab staff will be happy to use the existing quad-stamping page in Sequencescape, or whether we should add this step to the 384 Limber pipeline.

I will create a new story to add quad stamping to Limber.
